### ISSUE TITLE
ParseReact.Mutation.Set does not accept value with array which has null

### ISSUE THREAD

**Issue by kidwm** [ParsePlatform/ParseReact#60]
I can do this:
```js
item.set("array", [null, null, null]);
item.save();
```
but when I did:
```js
ParseReact.Mutation.Set(item, {array: [null, null, null]}).dispatch();
```
I received the error `Uncaught TypeError: Cannot read property 'objectId' of null`

BTW,
```js
ParseReact.Mutation.Set(item, {array: [undefined, undefined, undefined]}).dispatch();
```
it will work, and the saved value will be `[null, null, null]`

**Comment by kidwm** [comment]
this is still happened in 0.3.1
![parse-react-bug](https://cloud.githubusercontent.com/assets/822446/8100141/3e660e6c-103a-11e5-9e9b-9d1f5929d180.png)

**Comment by andrewimm** [comment]
Located the issue, someone will have a fix shortly.

**Comment by andrewimm** [comment]
Fixed in 0.3.2
